jerry-core/ecma/operations/ecma-proxy-object.c

```diff
@@ -1147,15 +1144,14 @@ ecma_proxy_object_get (ecma_object_t *obj_p, /**< proxy object */
         && !(target_desc.flags & ECMA_PROP_IS_WRITABLE)
         && !ecma_op_same_value (trap_result, target_desc.value))
     {
-      ret_value = ecma_raise_type_error (ECMA_ERR_MSG ("given property is a read-only and non-configurable"
-                                                       " data property on the proxy target"));
+      ret_value = ecma_raise_type_error (ECMA_ERR_MSG ("Incorrect value is returned by a Proxy 'get' trap"));
     }
     else if (!(target_desc.flags & ECMA_PROP_IS_CONFIGURABLE)
             && (target_desc.flags & (ECMA_PROP_IS_GET_DEFINED | ECMA_PROP_IS_SET_DEFINED))
             && target_desc.get_p == NULL
             && !ecma_is_value_undefined (trap_result))
     {
-      ret_value = ecma_raise_type_error (ECMA_ERR_MSG ("given property is a non-configurable property and"
+      ret_value = ecma_raise_type_error (ECMA_ERR_MSG ("Property of a Proxy is non-configurable and"
                                                        " does not have a getter function"));
```

Comment:
move the leading space from the string literal on this line to the end of the previous line (to make it consistent with other multi-line literals)